types/electron-devtools-installer/electron-devtools-installer-tests.ts

```diff
@@ -19,4 +12,7 @@ installExtension(REACT_PERF);
 installExtension(CYCLEJS_DEVTOOL);
 installExtension(APOLLO_DEVELOPER_TOOLS);
 installExtension(MOBX_DEVTOOLS);
-installExtension('abcdefghijkl');
\ No newline at end of file
+installExtension('abcdefghijkl');
+installExtension([EMBER_INSPECTOR, JQUERY_DEBUGGER]);
```

Comment:
As a side-request, while you're already covering Ember+jQuery and React+MobX extension installations with an array, can you simply remove installExtension(EMBER...), jQuery, React, MobX tools from above please (and see what happens)?
This way we can have much more meaningful test script here I believe.